lib/ClangImporter/ImportType.cpp

```diff
@@ -2252,114 +2403,37 @@ ParameterList *ClangImporter::Implementation::importFunctionParameterList(
     knownNonNull |= clangDecl->isFunctionTemplateSpecialization();
 
     // Check nullability of the parameter.
-    OptionalTypeKind OptionalityOfParam = getParamOptionality(param, knownNonNull);
+    OptionalTypeKind optionalityOfParam =
+        getParamOptionality(param, knownNonNull);
```

Comment:
Looks like this is the same logic twice. Can we move this inside of `importParameterType`?

Reply:
Yes. I will move it inside the function. My bad.

Reply:
I saw one problem: in `importMethodParamsAndReturnType` there's an extra usage of `importKind` outside the code that ends up in `importParameterType` (when completion handlers are dealt with) , so there will be an extra call, but hopefully that's fine.